CMakeLists.txt

```diff
@@ -109,13 +109,9 @@ if(NOT WASM)
     set(VULKAN_SHADER_DIR ${CMAKE_CURRENT_SOURCE_DIR}/src/Griddly/Core/Observers/Vulkan/resources/shaders)
 
     if(MSVC)
-        execute_process ( COMMAND ${VULKAN_SHADER_DIR}/compile_shaders.bat  
-                        WORKING_DIRECTORY ${VULKAN_SHADER_DIR} 
-                        RESULT_VARIABLE rv)
+        execute_process(COMMAND ./compile_shaders.bat RESULT_VARIABLE rv)
     else()
-        execute_process ( COMMAND bash ${VULKAN_SHADER_DIR}/compile_shaders.sh  
-                        WORKING_DIRECTORY ${VULKAN_SHADER_DIR} 
-                        RESULT_VARIABLE rv)
+        execute_process(COMMAND bash ./compile_shaders.sh RESULT_VARIABLE rv)
```

Comment:
What I am still wondering is, if the `.` is fine or should I replace it with `${CMAKE_CURRENT_SOURCE_DIR}`.

Reply:
I think keep it relative to the source directory because different build systems will call this script from different places and thus resolve the `.` to the wrong place.